config/config.json

```diff
@@ -21,7 +21,7 @@
         "EnablePostUsernameOverride": false,
         "EnablePostIconOverride": false,
         "EnableTesting": false,
-        "EnableDeveloper": false,
+        "EnableDeveloper": true,
```

Comment:
Definitely not intentional on your part, but we should not change `config.json` except to add new fields or change existing defaults.